core_api/src/routes/chat.py

```diff
@@ -45,13 +45,15 @@
 )
 
 ROUTE_RESPONSES = {
-    "info": ChatPromptTemplate.from_template(INFO_RESPONSE),
-    "ability": ChatPromptTemplate.from_template(ABILITY_RESPONSE),
-    "coach": ChatPromptTemplate.from_template(COACH_RESPONSE),
-    "gratitude": ChatPromptTemplate.from_template("You're welcome!"),
-    "retrieval": build_retrieval_chain,
-    "summarisation": build_summary_chain,
-    "extract": ChatPromptTemplate.from_template("You asking to extract some information - route not yet implemented"),
+    ChatRoute.info: ChatPromptTemplate.from_template(INFO_RESPONSE),
```

Comment:
Could we make these attributes of the `ChatRoute` objects?

Reply:
this is likely to be overwritten by https://github.com/i-dot-ai/redbox-copilot/pull/587